my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
There's no comments on this code. It will be great to comment each "box", explain what this does:

box 1 = import packages
box 2 = load fits image to a matrix variable

and so on...